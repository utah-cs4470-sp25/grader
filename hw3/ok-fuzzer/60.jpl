time show [ R0B_6I_ ]
time show [ q9k_ ]
show 88
print "{T`"
