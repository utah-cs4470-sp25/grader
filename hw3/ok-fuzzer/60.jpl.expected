(TimeCmd (ShowCmd (ArrayLiteralExpr (VarExpr R0B_6I_))))
(TimeCmd (ShowCmd (ArrayLiteralExpr (VarExpr q9k_))))
(ShowCmd (IntExpr 88))
(PrintCmd "{T`")
